unstructured/ingest/cli/cmds/elasticsearch.py

```diff
@@ -32,6 +31,45 @@ def get_cli_options() -> t.List[click.Option]:
                 help="If provided, will limit the fields returned by Elasticsearch "
                 "to this comma-delimited list",
             ),
+            click.Option(
```

Comment:
Would you know why none of these new ones show up with:
`unstructured-ingest elasticsearch --help`

Does a process have to run before this info is accessable? --batch-size is there.

Reply:
I can see them:
```shell
PYTHONPATH=. python ./unstructured/ingest/main.py elasticsearch --help
Usage: main.py elasticsearch [OPTIONS] COMMAND [ARGS]...ticsearch --help             

Options:
  --index-name TEXT               Name for the Elasticsearch index to pull
                                  data from  [required]
  --hosts DELIMITED-STRING        List of the Elasticsearch hosts to connect
                                  to, e.g. "http://localhost:9200"
  --fields DELIMITED-STRING       If provided, will limit the fields returned
                                  by Elasticsearch to this comma-delimited
                                  list
  --username TEXT                 username when using basic auth
  --password TEXT                 password when using basic auth or connecting
                                  to a cloud instance
...
```

Reply:
On same note, but not related to this pr. How do we get Weaviate in the --help destinations list?

Reply:
Are you not seeing it listed? When I run the same command as above I see this list: 
```shell
Destinations:
  azure
  azure-cognitive-search
  box
  delta-table
  dropbox
  fsspec
  gcs
  mongodb
  pinecone
  s3
  weaviate
```

Reply:
ok. interesting. is there something I have to run? `make doc` or something? I just have your branch loaded in git.